### PR TITLE
i143 3.10 map_variables informative

### PR DESCRIPTION
**Pull request by kerimoyle**
Addresses #141 
Rendered version is here: https://cellml-specification-dev.readthedocs.io/en/i143_3_10_map_variables_informative/reference/formal_and_informative/specC10_interpretation_of_map_variables.html

### CODE REVIEW

**Comment by kerimoyle** [comment]
Progress on this is waiting for issues #170, #171, #172, and #173 to be resolved.

File: src/reference/informative/informC10_interpretation_of_map_variables1.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    This is the goal of "variable equivalence", and requires the combined mechanisms of :code:`connection` elements (to connect components) and :code:`map_variables` elements (to connect their variables).
+    Several different words have been used to describe what is essentially this one process: equivalent variables, the connected variable set, mapped variables, etc.
+    All of these mean the same thing: that the variables in the set act as a single agent throughout the model.
+    They may have different local names within a component, but they will have the same value everywhere they exist.
+
+    Thus there are two ways in which variables are related to one another.  
+    The first is through the mathematical equations as specified in intra-component :code:`math` elements; the second is through variable equivalences as specified in inter-component :code:`connection` and :code:`map_variables` elements.
+
+    Both the :code:`connection` and :code:`map_variables` elements are *undirected*.
+    There is no update direction or hierarchy; they all have, instantaneously, the same value.
```
Comment (MichaelClerx) [inline src/reference/informative/informC10_interpretation_of_map_variables1.rst:25]: maybe go a step further than "have the same value" and say they represent the same physical thing?

File: src/reference/informative/informC10_interpretation_of_map_variables5.rst [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+      Understanding interfaces
+
+    There are two hurdles to get past before variables can be mapped to one another.
+    The first is the relative position within the encapsulation hierarchy of the :code:`variable` elements' parent :code:`component` elements (as discussed in :ref:`Interpretation of encapsulation<specC_interpretation_of_encapsulation>`, and the second is the value of the :code:`interface_type` attribute given to each :code:`variable` itself.
+    This means that even if two components are sufficiently close relatives that they *can* be connected, the modeller still has control at the individual variable level as to which of those relatives can have access.
+
+    The :code:`interface_type` attribute value choices are: 
+    - :code:`public` (used by child components to reference their parent, or between siblings),
+    - :code:`private` (used by parent components to reference their children), 
+    - :code:`public_and_private` (used by all near relatives), or
```
Comment (MichaelClerx) [inline src/reference/informative/informC10_interpretation_of_map_variables5.rst:22]: "used if both a public and a private interface is required"

File: src/reference/informative/informC10_interpretation_of_map_variables1.rst (hunk 1 above)
Comment (kerimoyle) [reply src/reference/informative/informC10_interpretation_of_map_variables1.rst]: Fixed in 9625b88

File: src/reference/informative/informC10_interpretation_of_map_variables5.rst (hunk 2 above)
Comment (kerimoyle) [reply src/reference/informative/informC10_interpretation_of_map_variables5.rst]: Fixed in 9625b88

File: src/reference/informative/informC10_interpretation_of_map_variables6.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    In :calvin_and_hobbes:`an episode of Bill Watterson's comic, Calvin and Hobbes <1990/01/09>`, Calvin builds a duplicator machine and clones himself several times so that the clones can do his chores in different rooms of the house.
+    This is essentially the purpose of creating :code:`map_variables` elements: to enable the "clones" (the mapped variables) to be used in separate "rooms" (components) whilst really representing the same thing.
+
+    In a *CellML model* there are two mechanisms by which variables can relate to one another: through the mathematics in the :code:`math` elements, and through the mapping in the :code:`map_variables` elements.
+    In a mathematical model, there is only one mechanism: the maths itself.
+    There are no clones, no duplicates, no mappings. 
+
+    Thus, the purely mathematical model is found by collapsing each set of mapped variables (cloned characters) onto a single unique variable (character), until the relationships between these unique variables no longer involve mapping (cloning) but only mathematics.
+    In the end, :calvin_and_hobbes"`Calvin does the same <1990/01/30>`, and the world returns to (roughly) normal.
```
Comment (MichaelClerx) [inline src/reference/informative/informC10_interpretation_of_map_variables6.rst]: " --> :

Comment (kerimoyle) [reply src/reference/informative/informC10_interpretation_of_map_variables6.rst]: Caught in a later commit I think? Maybe 4484eba?

**Pull request merged**